### DISCUSSION TITLE
4.1.3 release?

### DISCUSSION THREAD

**Discussion by lfcnassif** [sepinf-inc/IPED#1733] [General]
All devs,

I plan to release a 4.1.3 version this week, since we have important fixes done. This are the current release notes:
https://github.com/sepinf-inc/IPED/blob/4.1.x/ReleaseNotes.txt

I'm going to include #1706 and possibly #1693. Since I included some improvements, should we include other enhancements, like #1647 , #1287, #1636, any other? Opinions @tc-wleite, @hauck-jvsh, @patrickdalla?

**Comment by wladimirleite** [comment]
Great! 
Including #1647, #1287 and #1636 seems fine.

**Reply by lfcnassif** [reply]
OK, I'll do.

**Reply by lfcnassif** [reply]
Unfortunately I just realized #1647 and #1287 need localization of new WhatsApp Strings and I'm not sure if foreign collaborators will help us fast enough... So maybe they will be delayed...

**Reply by wladimirleite** [reply]
Indeed. We can postpone them to 4.2.

**Comment by patrickdalla** [comment]
I just pushed a PR to solve timeline synchronization on 4.1.x, as you
asked. I think you can include.

Em ter., 27 de jun. de 2023 15:24, Wladimir Leite ***@***.***>
escreveu:

**Reply by lfcnassif** [reply]
Great, thank you!

**Comment by hauck-jvsh** [comment]
Seems good to me.

**Comment by lfcnassif** [comment]
Since we are including some enhancements, should we also include #1525 and #1633?

**Comment by hauck-jvsh** [comment]
If it is because of it being a minor version, the #1633 changes more the behavior, but to me it is okay to include both.

**Reply by lfcnassif** [reply]
> the #1633 changes more the behavior

Good point!

**Comment by lfcnassif** [comment]
@tc-wleite I saw after merging #1651, you sent some fixing PRs. Could you help pointing which of them should I copy to 4.1.x?

**Reply by lfcnassif** [reply]
#1681 and #1682?

**Reply by wladimirleite** [reply]
Correct!
